bundles/model/org.eclipse.smarthome.model.lsp/src/main/java/org/eclipse/smarthome/model/lsp/internal/ModelServer.java

```diff
@@ -54,6 +56,7 @@
     static final String CONFIG_PID = "org.eclipse.smarthome.lsp";
     private static final String KEY_PORT = "port";
     private static final int DEFAULT_PORT = 5007;
+    private final ExecutorService pool = ThreadPoolManager.getPool("Language Server");
```

Comment:
We usually use camel case names starting lowercase - like e.g. "thingHandler". Maybe change this here to "languageServer" or simply "lsp"?